lib/ruby.test.rb

```diff
@@ -0,0 +1,49 @@
+
+def reverse_sentence(my_sentence)
```

Comment:
Nice in-place solution, notice you can create a helper method to reverse a section of a string and then reuse that, once to reverse the whole string and again to reverse each individual word.